src/ssh_audit/dheat.py

```diff
@@ -442,7 +443,7 @@ def _close_socket(socket_dict: Dict[socket.socket, float], s: socket.socket) ->
                 # out.d("Creating socket (%u of %u already exist)..." % (len(socket_dict), concurrent_sockets), write_now=True)
                 ret = s.connect_ex((target_ip_address, aconf.port))
                 num_attempted_connections += 1
-                if ret in [0, 115]:  # Check if connection is successful or EINPROGRESS.
+                if ret in [0, errno.EINPROGRESS]:  # Check if connection is successful or EINPROGRESS.
                     socket_dict[s] = now
                 else:
                     out.d("connect_ex() returned: %d" % ret, write_now=True)
```

Comment:
Maybe the error code should be translated to a string as well otherwise one has to consult `man errno` all the time.

Reply:
Switched, using `os.strerror`